pkg/kubelet/images/image_gc_manager_test.go

```diff
@@ -462,7 +462,9 @@ func TestFreeSpaceRemoveByLeastRecentlyUsed(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, manager.imageRecordsLen(), 2)
 
-	spaceFreed, err := manager.freeSpace(1024, time.Now())
+	// We're setting the delete time one minute in the future, so the time the image
```

Comment:
what's the granularity of the windows clock? should we inject sleeps between the calls to time.Now() above to ensure the changes are seen at at least that granularity?

Reply:
I'm trying to understand if the `detectImages` calls above that pass in time.Now() are depending on time.Now() being different between the calls to be flake free